lms/static/scripts/postmessage_json_rpc/server/test/index-test.js

```diff
@@ -0,0 +1,80 @@
+import { startRpcServer, getSidebarWindow, $imports } from '../index';
```

Comment:
@robertknight 
Was not sure what our standard was for testing index.js files, but I added these tests in any case.

Reply:
I suggest that in cases where we do write tests for a file named `index.js`, we treat it just like any other module.